### PR TITLE
Emagged Borg Sound Effect

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

- Emagged Borgs now play a Sound Effect

##Media

https://github.com/space-wizards/space-station-14/assets/113228053/bacd5fd5-7c7a-4413-a5a7-c31e61553580

**Changelog**
:cl:
- add: Emagging borgs now plays a sound effect.

### CODE REVIEW

File: Resources/Audio/Ambience/Antag/attributions.yml [hunk 1]
```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
+ - files: ["emag_borg.ogg"]
```
Comment (ficcialfaint) [inline Resources/Audio/Ambience/Antag/attributions.yml:13]:
```suggestion

- files: ["emag_borg.ogg"]
```

File: Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml [hunk 2]
```diff
@@ -58,6 +58,8 @@
     key: enum.BorgUiKey.Key
   - type: SiliconLawBound
   - type: EmagSiliconLaw
+    sound:
+      path: /Audio/Ambience/Antag/emag_borg.ogg
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml:62]: There is no such datafield in `EmagSiliconLaw`

**Review by ficcialfaint — CHANGES_REQUESTED** [review]
Test your changes locally before opening a PR

**Comment by EmoGarbage404** [comment]
this PR doesn't do anything? You need to actually implement the logic for playing the sound.

**Comment by Partmedia** [comment]
It looks like the logic is still missing, not to mention there are YAML lint problems. If you need help, please contact us in [#howdoicode on Discord](https://discord.com/channels/310555209753690112/560845886263918612).

**Comment by deltanedas** [comment]
the video isnt even from ingame bruh

**Comment by potato1234x** [comment]
you gotta actually write code for this you know. also sound is way too long

File: Content.Server/Silicons/Laws/SiliconLawSystem.cs [hunk 3]
```diff
@@ -37,12 +37,14 @@ public sealed class SiliconLawSystem : SharedSiliconLawSystem
     [Dependency] private readonly SharedStunSystem _stunSystem = default!;
     [Dependency] private readonly IEntityManager _entityManager = default!;
     [Dependency] private readonly SharedRoleSystem _roles = default!;
+    [Dependency] private readonly SharedAudioSystem _audioSystem = default!;
```
Comment (deltanedas) [inline Content.Server/Silicons/Laws/SiliconLawSystem.cs:40]: just adding a dependency doesnt make it play anything...

**Pull request closed without merging**